ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -653,4 +653,6 @@ SENTRY_API_KEY = '{{ localsettings.SENTRY_API_KEY }}'
 
 {% if localsettings.get('GREENHOUSE_API_KEY') %}
 GREENHOUSE_API_KEY = '{{ localsettings.GREENHOUSE_API_KEY }}'
-{% endif %}
\ No newline at end of file
+{% endif %}
+
+LOCAL_LOGGING_LOGGERS = {{ localsettings.get('LOCAL_LOGGING_LOGGERS', {}) }}
```

Comment:
Not sure if there is better way to disable 500 emails. Also, confirming that the `notify_exception` will still be logged in Sentry after this change